### PR TITLE
Cleanup renderers

### PR DESCRIPTION
**Pull request by alexanderGugel**

### CODE REVIEW

**Comment by michaelobriena** [comment]
@alexanderGugel this is very broad.  Can you factor this into smaller prs?

**Comment by alexanderGugel** [comment]
@michaelobriena The commits mostly heavily depend on each other. It might look like a big change, but it's really not that much. The commit messages have been updated to reflect the changes.

**Comment by michaelobriena** [comment]
This looks good.

File: core/Scene.js [hunk 1]
```diff
@@ -66,10 +66,6 @@ function Scene (selector, updater) {
     this.onMount(this, selector); // Mount the context to itself
                                   // (it is its own parent)
 
-    this._updater                  // message a request for the dom
```
Comment (alexanderGugel) [inline core/Scene.js]: No longer needed.

File: renderers/Compositor.js [hunk 2]
```diff
@@ -152,24 +152,6 @@ Compositor.prototype.getOrSetContext = function getOrSetContext(selector) {
...
- * @method
- * @private
- *
- * @param  {Number} iterator position index within the command queue
- * @param  {Array} commands remaining message queue received, used to
- * shift single messages
- *
- * @return {undefined} undefined
- */
-Compositor.prototype.giveSizeFor = function giveSizeFor(iterator, commands) {
```
Comment (alexanderGugel) [inline renderers/Compositor.js]: No longer needed.

**Comment by alexanderGugel** [comment]
@michaelobriena This removed DOMRenderer#getSize (redundant). IMO this has never been exposed to the user, but should this still be marked as a breaking change?

**Comment by michaelobriena** [comment]
@alexanderGugel Yea I would still consider it breaking.

**Comment by alexanderGugel** [comment]
Ok, will change.

**Comment by michaelobriena** [comment]
@alexanderGugel landed as e8a96d380527466ed6d75849ae3cdc7fb72dc294

**Pull request closed without merging**